x-pack/plugin/esql/qa/server/multi-clusters/src/javaRestTest/java/org/elasticsearch/xpack/esql/ccq/Clusters.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.esql.ccq;
+
+import org.elasticsearch.test.cluster.ElasticsearchCluster;
+import org.elasticsearch.test.cluster.local.distribution.DistributionType;
+import org.elasticsearch.test.cluster.util.Version;
+
+public class Clusters {
+    public static ElasticsearchCluster remoteCluster() {
+        return ElasticsearchCluster.local()
+            .name("remote_cluster")
+            .distribution(DistributionType.DEFAULT)
+            .version(Version.fromString(System.getProperty("tests.old_cluster_version")))
+            .nodes(2)
+            .setting("node.roles", "[data,ingest,master]")
+            .setting("xpack.security.enabled", "false")
+            .setting("xpack.license.self_generated.type", "trial")
+            .build();
```

Comment:
Of note, if all the tests in this project can share the same cluster you can use `.shared(true)` and that way the clusters won't be torn down after each test class. This makes execution a bit faster and more efficient.

Reply:
Cool, thanks. I've made the cluster shared in https://github.com/elastic/elasticsearch/pull/101640/commits/d9f66531664322d29924f01b78883a6e500e59a3